eigen_pod_proofs.go

```diff
@@ -103,6 +89,17 @@ func (epp *EigenPodProofs) ComputeBeaconStateTopLevelRootsCapella(beaconState *c
 	return beaconStateTopLevelRoots, err
 }
 
+func (epp *EigenPodProofs) ComputeVersionedBeaconStateTopLevelRoots(beaconState *spec.VersionedBeaconState) (*beacon.BeaconStateTopLevelRoots, error) {
```

Comment:
is this method expected to be called by future API users? If not, please make it private.